docs/DeveloperGuide.md

```diff
@@ -603,6 +606,101 @@ The following sequence diagram shows how the `login` command works:
+fields to be edited,`Name`, `Phone`, `Email` or/and `Address`.
+e.g.(`customer edit 1 --name John --phone 92149601`)
+2. If no fields are provided, an error message will prompt the user to key in at least one of the fields.
+3. If the customer id provided is negative or zero, an error message will prompt the user to key in an unsigned
+positive integer.
+4. The customer id provided is then cross-referenced with the stored customer list in `Model` to ensure that
+it corresponds to an existing `Customer`. If the customer id is not tied to any `Customer`, an error message will
+inform the user that that is the case.
+5. If the details provided exactly match the details of the `Customer` that was specified, an error message will inform
+   the user that the customer already exists in the address book.
+6. If all the previous steps are completed without exceptions, the fields of the `Customer` that was specified will
+be updated with the new information provided by the user.
+
+The following activity diagram shows the logic of a user editing a customer's information:
+
+<puml src="diagrams/CustomerEditActivityDiagram.puml" alt="CustomerEditActivityDiagram" />
+
+The sequence of the `customer edit` command is as follows:
+
+1. The user inputs the `customer edit` command with `input` as the customer id and `Name`, `Phone`, `Email` and/or
+`Address` as the fields to be edited. e.g.(`customer edit 1 --name John --phone 92149601)
+2. The `LogicManager` calls `AddressBookParser#parseCommand` to create its corresponding CommandParser.
+3. In this case, the `AddressBookParser` creates an instance of `CustomerEditCommandParser` and calls
+`CustomerEditCommandParser#parse` to parse the given `input` using various parse methods from `ParserUtil` and
+   creates a `CustomerEditDescriptor` object.
+4. The `CustomerEditCommandParser` then creates a `CustomerEditCommand` object. The `CustomerEditCommand` object
+takes in the `CustomerEditDescriptor` instance and the customer id from Step 1 as a parameter.
+5. The `CustomerEditCommand` is then returned to the `LogicManager` where its execute method is called. This creates
+a `Customer` object by calling `CustomerEditCommand#createEditedCustomer`. Also, it edits the `Customer` with the
+   customer id input in Step 1. This is done by calling `Model#setCustomer`.
+6. With the `Customer` specified edited, a `CommandResult` with a success message is then returned.
+
+The following sequence diagram shows how the `customer edit` command works:
+
+<puml src="diagrams/CustomerEditSequenceDiagram.puml" alt="CustomerEditSequenceDiagram" />
+
+### Delivery Add Command
+
+**Overview:**
+
+The `delivery add` command is used to add a new Delivery with all the given information fields
+specified by the user, namely the delivery's `DeliveryName`, customer id of a `Customer` and `DeliveryDate`. All 
+fields are compulsory.
+
+The format for the `delivery add` command can be found [here](UserGuide.md#login).
```

Comment:
Same as above, need to update the UserGuide link.

Reply:
Done